sdk/core/azure-core-amqp/test/ut/management_tests.cpp

```diff
@@ -595,7 +586,6 @@ namespace Azure { namespace Core { namespace Amqp { namespace Tests {
 
     mockServer.StopListening();
 #else
-    EXPECT_TRUE(false);
 #endif
```

Comment:
nit: This is now an empty #else/#endif block. Remove #else, and just keep #if/#endif

Reply:
Keeping the #else is intentional for now. I will be handling the management tests for rust separately when I get around to cleaning up all the AMQP level tests (the C# AMQP test broker that the other languages use doesn't support management API tests).